examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
Looks like I may have been suffering from sleep deprivation as now the Bind overall color seems to be working. 

I'll get the shaders changed and retest gles2 and 3 thanks for the help